src/muya/lib/utils/index.js

```diff
@@ -166,6 +166,8 @@ export const checkImageContentType = url => {
         } else {
           settle(false)
         }
+      } else if (req.status === 405) { // status 405 means method not allowed, and just return true
+        settle(true)
```

Comment:
I think you should reference to the issue, so that we later know why we did this :smile:

Reply:
Ok!